internal/docs/readme.go

```diff
@@ -17,96 +17,116 @@ import (
 	"github.com/elastic/elastic-package/internal/packages"
 )
 
-// ReadmeFile for the Elastic package
-const ReadmeFile = "README.md"
-
-// IsReadmeUpToDate function checks if the README file is up-to-date.
+// IsReadmeUpToDate function checks if all .md readme files are up-to-date.
 func IsReadmeUpToDate() (bool, error) {
```

Comment:
In keeping with https://github.com/elastic/elastic-package/pull/282/files#r591254461, I supposed this function might need to be updated to return a list of files that need updating?

Reply:
Yep! Changed it to report a list of files :) Thanks!!